harper-core/src/linting/closed_compounds.rs

```diff
@@ -0,0 +1,342 @@
+create_closed_compound_linter!(Worldwide, "world wide", "worldwide");
+create_closed_compound_linter!(Overall, "over all", "overall");
+create_closed_compound_linter!(Likewise, "like wise", "likewise");
+create_closed_compound_linter!(However, "how ever", "however");
+create_closed_compound_linter!(Altogether, "all together", "altogether");
+create_closed_compound_linter!(Asleep, "a sleep", "asleep");
+create_closed_compound_linter!(Upset, "up set", "upset");
+create_closed_compound_linter!(Intact, "in tact", "intact");
+create_closed_compound_linter!(Somehow, "some how", "somehow");
+create_closed_compound_linter!(Proofread, "proof read", "proofread");
+create_closed_compound_linter!(Nobody, "no body", "nobody");
+create_closed_compound_linter!(Somebody, "some body", "somebody");
+create_closed_compound_linter!(Anybody, "any body", "anybody");
+create_closed_compound_linter!(Nothing, "no thing", "nothing");
+create_closed_compound_linter!(Anywhere, "any where", "anywhere");
+create_closed_compound_linter!(Instead, "in stead", "instead");
+create_closed_compound_linter!(Somewhere, "some where", "somewhere");
+create_closed_compound_linter!(Middleware, "middle ware", "middleware");
+create_closed_compound_linter!(Into, "in to", "into");
+create_closed_compound_linter!(Overclocking, "over clocking", "overclocking");
+create_closed_compound_linter!(Backplane, "back plane", "backplane");
+create_closed_compound_linter!(Overload, "over load", "overload");
+create_closed_compound_linter!(Underclock, "under clock", "underclock");
+create_closed_compound_linter!(Devops, "dev ops", "devops");
+create_closed_compound_linter!(Multithreading, "multi threading", "multithreading");
+create_closed_compound_linter!(Everywhere, "every where", "everywhere");
+create_closed_compound_linter!(Multicore, "multi core", "multicore");
+create_closed_compound_linter!(Multimedia, "multi media", "multimedia");
+create_closed_compound_linter!(Ecommerce, "e commerce", "ecommerce");
+create_closed_compound_linter!(Widespread, "wide spread", "widespread");
+create_closed_compound_linter!(Notwithstanding, "not with standing", "notwithstanding");
+create_closed_compound_linter!(Anyhow, "any how", "anyhow");
+create_closed_compound_linter!(Nonetheless, "none the less", "nonetheless");
+create_closed_compound_linter!(Hereafter, "here after", "hereafter");
+create_closed_compound_linter!(Otherwise, "other wise", "otherwise");
+create_closed_compound_linter!(Therein, "there in", "therein");
+create_closed_compound_linter!(Thereupon, "there upon", "thereupon");
+create_closed_compound_linter!(Hereby, "here by", "hereby");
+create_closed_compound_linter!(Hereunder, "here under", "hereunder");
+create_closed_compound_linter!(Forthwith, "forth with", "forthwith");
+create_closed_compound_linter!(Insofar, "in so far", "insofar");
+create_closed_compound_linter!(Whereupon, "where upon", "whereupon");
+create_closed_compound_linter!(Thereafter, "there after", "thereafter");
```

Comment:
"I got there after him". "Go back there after dinner and finish it." etc. Much more common than the legalese.